src/styles/_project-finsec.scss

```diff
@@ -115,7 +114,7 @@
     }
   }
   #hero2 {
-    background-image: url('/assets/img/portfolio/finsec/background.jpg');
+    background-color: #1A183A;
```

Comment:
@richardverheyen Why are we using 38kb image where we can simply use a `background-color`??!